example/lib/main.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_chat_ui/flutter_chat_ui.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Chat Ui Demo',
+      theme: ThemeData(primarySwatch: Colors.blue),
+      home: const MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({Key? key}) : super(key: key);
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  final _scrollController = ScrollController();
+  final _list = [
+    Message(
+      id: '1',
+      message: "Hi",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+    ),
+    Message(
+      id: '2',
+      message: "Hi",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+    ),
+    Message(
+      id: '3',
+      message: "How are you?",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+    ),
+    Message(
+      id: '4',
+      message: "I am fine",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+    ),
+    Message(
+      id: '5',
+      message: "Okay",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+      reaction: '❤️',
+    ),
+    Message(
+      id: '6',
+      message: "Do you hav idea about this?",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+    ),
+    Message(
+      id: '7',
+      message:
+          "https://docs.flutter.dev/development/packages-and-plugins/developing-packages",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+      replyMessage: ReplyMessage(
+          message: "Do you hav idea about this?", replyTo: "Faiyaz"),
+    ),
+    Message(
+      id: '8',
+      message:
+          "https://images.pexels.com/photos/586744/pexels-photo-586744.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+    ),
+    Message(
+      id: '9',
+      message:
+          "https://images.pexels.com/photos/586744/pexels-photo-586744.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+      reaction: '❤️',
+    ),
+    Message(
+      id: '6',
+      message: "EveryThing is fine?",
+      createdAt: DateTime.now(),
+      sendBy: "Faiyaz",
+    ),
+    Message(
+      id: '6',
+      message: "😂😂",
+      createdAt: DateTime.now(),
+      sendBy: "Dhvanit",
+    ),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Flutter Chat Ui Demo'),
+      ),
+      body: ChatList(
+        scrollController: _scrollController,
+        messageList: _list,
+        currentUser: "Dhvanit",
+        backgroundImageFromAsset: "assets/images/background-image.jpeg",
+        outgoingChatBubbleBorderRadius:
```

Comment:
instead of the separate parameter of outgoing and incoming chat bubbles make a class for their configuration.